asciidoc/client-applications.adoc

```diff
@@ -90,37 +94,48 @@ include::{javascript-examples}/examples.test.js[tags=driver-lifecycle]
 // ----
 ======
 
-// [.include-with-python]
-// ======
-// [source, python]
-// ----
-// include::{python-examples}/driver_lifecycle_example.py[tags=driver-lifecycle]
-// ----
-// ======
+[.include-with-python]
+======
+[source, python]
+----
+include::{python-examples}/test_driver_lifecycle_example.py[tags=driver-lifecycle]
+----
+[source, python]
+----
+include::{python-examples}/test_driver_lifecycle_example.py[tags=driver-lifecycle-import]
+----
+======
 ====
 
 
 [[driver-connection-uris]]
 == Connection URIs
 
-A connection URI identifies a graph database and how to connect to it.
-The encryption and trust settings provide detail to how that connection should be secured.
+A connection URI *_identifies a graph database and how to connect to it_*.
+
+The *_encryption_* and *_trust_* settings provide detail to how that connection should be secured.
 
 [CAUTION]
-There are significant changes to security settings between Neo4j 3.x and Neo4j 4.x.
+====
+*There are significant changes to security settings between Neo4j 3.x and Neo4j 4.x !*
```

Comment:
I'm not sure if the added emphasis is need in this sentence, as it's already in a warning admonition. We also try to avoid using exclamation marks in the documentation.